components/CTA.tsx

```diff
@@ -1,72 +1,78 @@
-  },
-  ["BUY_GLO_MERCH"]: {
-    title: "Buy Glo Merch",
-    description:
-      "Glo is meant to be spent. Visit the Glo store and order a hoodie!",
-    iconPath: "/buy.svg",
-    url: "https://merch.glodollar.org",
-    // action: () => open a modal in the future,
-  },
-  ["JOIN_PROGRAM"]: {
-    title: "Join as early adopter",
-    description: "Be the change you want to see in the world.",
-    iconPath: "/za-warudo.svg",
-    // action: () => open a modal in the future,
-    url: "https://www.glodollar.org/get-started",
-  },
-};
-
-const ActionButton = ({ ctaType, idx }: { ctaType: CTAType; idx: number }) => {
-  const cta = CTAs[ctaType];
+const ActionButton = ({
+  CTA_MAP,
+  email,
+  ctaType,
+}: {
+  CTA_MAP: { [key in CTAType]: ActionButton };
+  email: string | undefined;
+  ctaType: CTAType;
+}) => {
+  const cta = CTA_MAP[ctaType];
   return (
-    <li key={`CTA${ctaType}`}>
-      <a
-        className={`flex cursor-pointer items-center py-4 ${
-          idx === Object.keys(CTAs).length ? "" : "border-b-2"
-        }`}
-        href={cta.url}
-        target="_blank"
-        rel="noreferrer"
-      >
-        <div className="mr-4 flex border justify-center min-w-[32px] min-h-[32px] rounded-full bg-pine-200">
-          <Image
-            src={cta.iconPath}
-            width={16}
-            height={16}
-            alt="call to action"
-          />
-        </div>
-        <div className="flex-col w-56">
-          <h2>{cta.title}</h2>
-          <p className="font-thin text-sm text-pine-700 leading-6">
-            {cta.description}
-          </p>
-        </div>
-        <Image
-          src="/arrow-right.svg"
-          width={25}
-          height={25}
-          alt="arrow-right"
-          className="ml-2 flex w-25px max-w-25px h-25px max-h-25px"
-        />
-      </a>
-    </li>
+    <button
+      className={`flex cursor-pointer items-center py-4 ${
+        ctaType ? "" : "border-b-2"
+      }`}
+      onClick={() => cta.action(email)}
+    >
+      <div className="mr-4 flex border justify-center min-w-[32px] min-h-[32px] rounded-full bg-pine-200">
+        <Image src={cta.iconPath} width={16} height={16} alt="call to action" />
+      </div>
+      <div className="flex-col w-56">
+        <h2>{cta.title}</h2>
+        <p className="font-thin text-sm text-pine-700 leading-6">
+          {cta.description}
+        </p>
+      </div>
+      <Image
+        src="/arrow-right.svg"
+        width={25}
+        height={25}
+        alt="arrow-right"
+        className="ml-2 flex w-25px max-w-25px h-25px max-h-25px"
+      />
+    </button>
   );
 };
 
 export default function CTA() {
   const { ctas } = useUserStore();
+  const email = Cookies.get("glo-email");
+  const CTA_MAP: { [key in CTAType]: ActionButton } = {
+    ["SHARE_GLO"]: {
+      title: "Share Glo with friends",
+      iconPath: "/megahorn.svg",
+      description: "Tell even more friends. Share your invite link.",
```

Comment:
The CTA texts have been changed. Please make sure you use the CTA texts from main, not these old ones.

Reply:
had a 50/50 on the merge and failed, fixing